metadata.tvshows.themoviedb.org.python/resources/language/resource.language.zh_tw/strings.po

```diff
@@ -0,0 +1,153 @@
+# Kodi Media Center language file
+# Addon Name: The Movie Database
+# Addon id: metadata.themoviedb.org
+# Addon Provider: XBMC Foundation
+msgid ""
+msgstr ""
+"Project-Id-Version: XBMC Addons\n"
+"Report-Msgid-Bugs-To: alanwww1@xbmc.org\n"
+"POT-Creation-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Automatically generated\n"
+"Language-Team: none\n"
+"Language: zh_tw\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+
+msgctxt "Addon Summary"
+msgid "Fetch TV Show metadata from themoviedb.org"
+msgstr ""
```

Comment:
This language doesn't seem to actually have translations?

Reply:
@razzeee All languages need to be created empty before they can be translated at Weblate.
We have a lot of untranslated languages for metadata.tvshows.themoviedb.org.
It's perfectly safe to add these languages, as all untranslated strings will fallback to en_gb.